ViAn/Filehandler/project.h

```diff
@@ -15,19 +15,20 @@
 #include "video.h"
 #include "saveable.h"
 typedef int ID;
+class FileHandler;
 
 /**
  * @brief The Project struct
  * incomplete struct, will be added on
  * along with parser functionality
  */
-
 struct Project : Saveable{
     std::string name;
-public:
-    Project();
-    Project(ID id, std::string name);
-    Project(std::string dir_path);
+    FileHandler* file_handler;
+public:    
+    Project(FileHandler* file_handler);
+    Project(FileHandler* file_handler, ID id, std::string name);
+    Project(FileHandler* file_handler, std::string dir_path);
```

Comment:
There's no implementation for this constructor `Project(FileHandler* file_handler, std::string dir_path)`.